### ISSUE TITLE
Messes with AutoHotkey shortcuts

### ISSUE THREAD

**Issue by anghelos** [damienleroy/PowerAccent#27]
I've made an AutoHotkey script to easily insert non-breaking spaces (`Ctrl`+`Shift`+`SPACE`).
When Quick Accent is enabled, this shortcut either doesn't do anything, or inserts a 0.
I've tried using only Left/Right Arrow as an activation key, with no changes. I've also changed my shortcut to avoid using the Space key, with no changes either.

Here's the AutoHotkey script, for reference:
```
#NoEnv
SendMode Input 
SetWorkingDir %A_ScriptDir%
^+SPACE:: send {Asc 0160}
```

**Comment by damienleroy** [comment]
Indeed, I can reproduce the bug. Thanks for the feedback, we will take care of that.

**Comment by anghelos** [comment]
It seems to have been fixed with the latest Powertoys release (0.62.1)
